katdal/ms_extra.py

```diff
@@ -39,7 +39,7 @@
     import casac
     tb = casac.homefinder.find_home_by_name('tableHome').create()
     casacore_binding = 'casapy'
-except:
+except Exception:
```

Comment:
I think there were other exceptions, since I'm generally not a `try...except:` kinda guy (and the pyrap leg has `ImportError` as reference).

On the other hand, the `casapy` binding has probably not been used since 2009 (or maybe 2013). I could always just rip it out too (eventually - this PR is getting off-topic :-)).

Reply:
Made this ImportError for now.